$OpenBSD$

Update the simplexmq Cabal bounds to the upstream-resolved versions used by
the port on OpenBSD.

--- ../simplexmq-stable/simplexmq.cabal
+++ ../simplexmq-stable/simplexmq.cabal
@@ -312,35 +312,35 @@
     , base >=4.14 && <5
     , base64-bytestring >=1.0 && <1.3
     , composition ==1.0.*
-    , constraints >=0.12 && <0.14
+    , constraints >=0.12 && <0.15
     , containers ==0.6.*
     , crypton ==0.34.*
     , crypton-x509 ==1.7.*
     , crypton-x509-store ==1.6.*
     , crypton-x509-validation ==1.6.*
     , cryptostore ==0.3.*
-    , data-default ==0.7.*
+    , data-default ==0.8.*
     , directory ==1.3.*
     , filepath ==1.4.*
     , hourglass ==0.2.*
     , http-types ==0.12.*
-    , http2 >=4.2.2 && <4.3
+    , http2 >=5.1 && <5.5
     , iproute ==1.7.*
     , iso8601-time ==0.1.*
     , memory ==0.18.*
     , mtl >=2.3.1 && <3.0
     , network >=3.1.2.7 && <3.2
     , network-info ==0.2.*
-    , network-transport ==0.5.6
+    , network-transport ==0.5.7
     , network-udp ==0.0.*
     , random >=1.1 && <1.3
-    , scientific ==0.3.7.*
+    , scientific ==0.3.8.1
     , simple-logger ==0.1.*
     , socks ==0.6.*
     , stm ==2.5.*
     , time ==1.12.*
-    , time-manager ==0.0.*
-    , tls >=1.9.0 && <1.10
+    , time-manager >=0.2 && <0.3
+    , tls >=2.0 && <2.3
     , transformers ==0.6.*
     , unliftio ==0.2.*
     , unliftio-core ==0.2.*
@@ -354,14 +354,14 @@
         case-insensitive ==1.2.*
       , hashable ==1.4.*
       , ini ==0.4.1
-      , optparse-applicative >=0.15 && <0.17
+      , optparse-applicative >=0.15 && <0.19
       , process ==1.6.*
       , temporary ==1.3.*
       , wai >=3.2 && <3.3
       , wai-app-static >=3.1 && <3.2
-      , warp ==3.3.30
-      , warp-tls ==3.4.7
-      , websockets ==0.12.*
+      , warp ==3.4.10
+      , warp-tls ==3.4.10
+      , websockets >=0.12 && <0.14
       , zlib >=0.6 && <0.8
   if flag(client_postgres) || flag(server_postgres)
     build-depends:
@@ -381,7 +381,7 @@
   if impl(ghc >= 9.6.2)
     build-depends:
         bytestring ==0.11.*
-      , template-haskell ==2.20.*
+      , template-haskell >=2.20 && <2.23
       , text >=2.0.1 && <2.2
   if impl(ghc < 9.6.2)
     build-depends:
